tests/unit/test_pytest_operator.py

```diff
@@ -283,9 +330,8 @@ async def test_crash_dump_mode(monkeypatch, tmp_path_factory):
     """Test running juju-crashdump in OpsTest.cleanup."""
     patch = monkeypatch.setattr
     patch(plugin.OpsTest, "run", mock_run := AsyncMock(return_value=(0, "", "")))
-    ops_test = plugin.OpsTest(
-        mock_request := Mock(**{"module.__name__": "test"}), tmp_path_factory
-    )
+    mock_request = Mock(**{"module.__name__": "test"})
+    ops_test = plugin.OpsTest(mock_request, tmp_path_factory)
```

Comment:
technically this was a valid use of the walrus operator -- but the linter doesn't care for it